client/landing/stepper/declarative-flow/anchor-fm-flow.ts

```diff
@@ -50,7 +50,7 @@ export const anchorFmFlow: Flow = {
 				case 'podcastTitle':
 					return navigate( 'designSetup' );
 				case 'designSetup':
-					return redirect( `/page/home/${ siteSlug }` );
+					return redirect( `/page/${ siteSlug }/home` );
```

Comment:
Sort of unrelated to this change, but I think `goNext()` for the `designSetup` step should redirect to the `processing` step rather than the editor here.